setup.py

```diff
@@ -64,6 +64,9 @@ def get_git_sha() -> str:
     zip_safe=False,
     entry_points={
         "console_scripts": ["superset=superset.cli.main:superset"],
+        "sqlalchemy.dialects": [
```

Comment:
```suggestion
        # the `postgres+psycopy2://` scheme was removed in SQLAlchemy 1.4, add an alias here
        # to prevent breaking existing databases
        "sqlalchemy.dialects": [
```